about.html

```diff
@@ -1,64 +1,108 @@
-
-  <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
-
-</head>
-
-<body>
+  <head>
+    <meta charset="utf-8" />
+
+    <title>Sprint Challenge - About</title>
+
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto|Rubik"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="css/index.css" />
+  </head>
+  <header>
+    <img src="img/lambda-black.png" alt="Lambda logo" />
+    <nav id="nav-bar">
+      <a href="index.html">Home</a>
+      <a href="about.html">About</a>
+      <a href=" ">Products</a>
+      <a href=" ">Blog</a>
+      <a href=" ">Contact</a>
+    </nav>
+  </header>
+
+  <body>
     <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-
-
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-working.png" alt="strategy">
-    
-    How We Work
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-office.png" alt="strategy">
-    
-    Places We Work
-
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-    <img src="img/about-meeting.png" alt="strategy">
-    
-    Collaboration
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-    Let's Work Together
+      <div class="first-section">
+        <img src="img/jumbo-about.png" />
+        <div class="text">
+          <h3>About</h3>
+
+          <p>
+            Leverage agile frameworks to provide a robust synopsis for high
+            level overviews. Iterative approaches to corporate strategy foster
+            collaborative thinking to further the overall value proposition.
+            Organically grow the holistic world view of disruptive innovation
+            via workplace diversity and empowerment.
+          </p>
+        </div>
+      </div>
+
+      <section class="top-categories">
+        <div class="category">
+          <img src="img/about-plan.png" alt="strategy" />
+
+          <h3>Strategy</h3>
+
+          <p>
+            Iterative approaches to corporate strategy foster collaborative
+            thinking to further the overall value proposition.
+          </p>
+
+          <button>Learn More</button>
+        </div>
+
+        <div class="category">
```

Comment:
When copying and pasting code remember to change any attributes that should change. The alt="strategy" is on all of these images.